.kiro/specs/brand-masthead/design.md

```
@@ -0,0 +1,245 @@
+
+## Architecture Overview
+
+```
+┌─────────────────────────────────────────────────────────────┐
+│                    MastheadSkia Component                    │
+├─────────────────────────────────────────────────────────────┤
+│  Props: { height, variant }                                  │
+│                                                              │
+│  ┌─────────────────────────────────────────────────────────┐│
+│  │                    Skia Canvas                          ││
+│  │  ┌──────────┐ ┌──────────┐ ┌──────────┐ ┌───┬───┬───┐  ││
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)